certified-connectors/Vineforce/apiDefinition.swagger.json

```diff
@@ -0,0 +1,204 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Vineforce",
+    "description": "Vineforce is a remote workplace tool built for Microsoft 365 users. It transforms the platform into one cohesive toolkit to help you get more done, experience less burnout, and feel more connected with your teammates.",
+    "version": "1.0",
+	"contact": {
+      "name": "Contact",
+      "url": "https://www.vineforce.com/contact",
+      "email": "contact@vineforce.com"
+    }
+  },
+  "host": "appapi.vineforce.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/services/app/ExternalTask/CreateExternalTask": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@nikunjvf Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
Updated with latest commit